GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitRepoTests.cs

```diff
@@ -215,6 +215,24 @@ protected void EditFile(string filePath, string content)
             this.FileSystem.AppendAllText(controlFile, content);
         }
 
+        protected void CreateHardLink(string targetPath, string newLinkPath)
+        {
+            string virtualTargetFile = Path.Combine(this.Enlistment.RepoRoot, targetPath);
+            string controlTargetFile = Path.Combine(this.ControlGitRepo.RootPath, targetPath);
+            string virtualNewLinkFile = Path.Combine(this.Enlistment.RepoRoot, newLinkPath);
+            string controlNewLinkFile = Path.Combine(this.ControlGitRepo.RootPath, newLinkPath);
+
+            // GitRepoTests are only run with SystemIORunner (which does not support hardlink
+            // creation) so use a BashRunner instead.
+            this.FileSystem.ShouldBeOfType<SystemIORunner>();
+            this.FileSystem.SupportsHardlinkCreation.ShouldBeFalse(
+                "FileSystem *does* support hard link creation, CreateHardLink no longer needs to create a BashRunner");

```

Comment:
This message is confusing. You're saying that if in the future it ends up supporting hardlinks, we can remove this code?

Reply:
Yep that's right, let me try re-working this message.